python/ray/serve/_private/deployment_state.py

```diff
@@ -1478,10 +1498,20 @@ def autoscale(self, current_handle_queued_queries: int):
         if decision_num_replicas == self._target_state.num_replicas:
             return
 
+        # Adjust the logged replica counts based on `target_capacity`. In reality the
+        # autoscaling algorithm operates on the full replica count and the target
+        # will be adjusted in the in the main deployment update cycle, but this log
+        # statement needs to be comprehensible to the user.
+        adjusted_curr_num_replicas = self.get_capacity_adjusted_num_replicas(
+            self._target_state.num_replicas, target_capacity
+        )
+        adjusted_decision_num_replicas = self.get_capacity_adjusted_num_replicas(
+            decision_num_replicas, target_capacity
+        )
         logger.info(
             f"Autoscaling replicas for deployment {self.deployment_name} in "
-            f"application {self.app_name} from {self._target_state.num_replicas} to "
-            f"{decision_num_replicas}. Current ongoing requests: "
+            f"application {self.app_name} from {adjusted_curr_num_replicas} to "
```

Comment:
> from {adjusted_curr_num_replicas}

If `target_capacity` was changed just before this iteration, then the `adjusted_curr_num_replicas` may not match the actual number of currently running replicas. I think we should use the `num_replicas` directly in this log message.

Reply:
This is a good point. Perhaps it's best to just leave off the "from" portion of the log message? I think it'll be pretty confusing if the autoscaler says we're scaling to 8 replicas but then we are happy at 4 due to a target_capacity of 50.

Reply:
Yeah, I'm okay with that.